electron/src/helpers/BigGraph.tsx

```diff
@@ -0,0 +1,639 @@
+        if (viewMode === 'default') {
+            setDefaultView();
+        }
+    };
+
+    // Get the display value: cursor value if hovering, otherwise latest value
+    const displayValue = cursorValue !== null ? cursorValue : newData?.current?.value;
+
+    // Get the current time window label for the button
+    const currentTimeWindowLabel = TIME_WINDOW_OPTIONS.find(option => option.value === selectedTimeWindow)?.label || '1 min';
+
+    return (
+        <div style={{ width: "100%", height: "100%", display: "flex", flexDirection: "column" }}>
+            {/* Control Panel - Outside the graph */}
+            <div
+                style={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                    alignItems: "center",
+                    padding: "8px",
+                    borderBottom: "1px solid #eee",
+                    backgroundColor: "#f9f9f9",
+                    flexShrink: 0,
+                }}
+            >
+                {/* Current Value Display - now shows cursor value when hovering */}
+                <div
+                    style={{
+                        background: "rgba(255, 255, 255, 0.9)",
+                        border: "1px solid #ccc",
+                        padding: "6px 12px",
+                        borderRadius: "4px",
+                        fontSize: "14px",
+                        fontWeight: "bold",
+                    }}
+                >
+                    Current: {displayValue !== undefined && displayValue !== null ? (renderValue ? renderValue(displayValue) : displayValue.toFixed(3)) : 'N/A'} {renderUnitSymbol(unit)}
+                </div>
+
+                {/* Control Buttons Container */}
+                <div style={{ display: "flex", gap: "8px", alignItems: "center" }}>
+                    {/* Time Window Selector */}
+                    <div style={{ display: "flex", alignItems: "center", gap: "4px" }}>
+                        <label style={{ fontSize: "12px", fontWeight: "500" }}>
+                            Time:
+                        </label>
+                        <select
+                            value={selectedTimeWindow}
+                            onChange={handleTimeWindowChange}
+                            style={{
+                                ...buttonStyle,
+                                backgroundColor: 'white',
+                                color: 'black',
+                                minWidth: '70px',
+                            }}
+                        >
+                            {TIME_WINDOW_OPTIONS.map(option => (
+                                <option key={option.value} value={option.value}>
+                                    {option.label}
+                                </option>
+                            ))}
+                        </select>
+                    </div>
+
+                    {/* Excel Export Button */}
+                    <button
```

Comment:
Please use `TouchButton` so it has a good size, maybe we can make it a bit smaller.